### ISSUE TITLE
Introduction too wordy

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-policies-prototype#13]
Think the first paragraph and last bullet-list in this section could be safely dropped. If needed the first paragraph could be made collapsible under a different heading, like "background" - people probably do not want/need to read this every time. The remaining paragraph + sentence in this section could be made more succinct too.

**Comment by bakkenb** [comment]
Agree with all recommendations from Shadi. Would like to know from Mary Jo however, what was her rationale for including the first paragraph.

Agree that this could be written more succinctly.

**Comment by nitedog** [comment]
I believe this is just a carry-over from the current page - not wording from MaryJo.

**Comment by slhenry** [comment]
+1

**Comment by maryjom** [comment]
@nitedog Yes, I carried over the front matter from the current page and edited it down for length and to update content only. That's why I had brought that up to the group to query if we could get rid of it, move it to the bottom or what to do to reduce the space taken by stuff that is not the real purpose of the content on the page.

**Comment by yatil** [comment]
I think if we can get rid of it, let’s get rid of it – if we have the structure and content organization on the page right, it might be obvious.

“UI is like a joke – if you have to explain it, it’s not so good.”

**Comment by AndrewArch** [comment]
seems partly related to #14 - possibly consider to group with the terminology and any other background on a separate 'help' or 'about' sub-page

**Comment by slhenry** [comment]
I like the idea of putting things together, but prefer collapsible &/or side bar, rather than separate sub-page. (Like on [WCAG-EM Report Tool](https://www.w3.org/WAI/eval/report-tool/#/))

**Comment by slhenry** [comment]
Current prototype has 4 headings (Disclaimer, Updates, Intro, Terms) for 5 sentences. Probably these can all go under one heading of Introduction or such.

(Also, I have minor copy edit suggestions I'll try to get to soon.)

**Comment by maryjom** [comment]
I've tried understanding the code of the prototype and have no idea how to combine these sections into a single one as this is not coded with plain HTML and are in 4 separate files with no real heading markup that I can see.  I'll assign to Robert, as I'm sure he will know how to do this.

**Comment by bakkenb** [comment]
I  have no issues with the current wording in the Introduction section now that it has been updated. I do have one comment however, maybe this will/would all be worked out in the live version anyway, but wanted to add it here. 

In the [intro sentence](https://w3c.github.io/wai-policies-prototype/#developing) it states...
> The Web Policy Overview table provides a quick reference of the various laws and policies used throughout the world. 

If we are going to refer to the table as the "Web Policy Overview" table, then I think that those words should appear in bold as a table title above the actual table further down the page.

**Comment by slhenry** [comment]
Proposal:

This page lists some governmental policies related to web accessibility, although it is not a comprehensive or definitive listing.

The information on this page is not legal advice. If you have questions about the applicability of the policies to specific situations, please consult legal authorities for the appropriate jurisdiction.

For guidance on developing an accessibility policy for an organization, see [Developing Organizational Policies on Web Accessibility](@@).

We welcome additions and corrections: [[Submit an Update]](@@)

**Comment by slhenry** [comment]
+ terms described with filtering options per https://github.com/w3c/wai-policies-prototype/issues/14#issuecomment-290767284

**Comment by bakkenb** [comment]
+1 to proposal.

**Comment by nrhsinclair** [comment]
I did not see the wording of the introduction before it was changed, but coming to this section without much background in the subject, to me some sort of introduction or background would be helpful before mentioning the table. Perhaps changing the order of first and second paragraphs to: 

Developing Organizational Policies on Web Accessibility addresses issues which frequently arise when establishing policies on Web accessibility.

The Web Policy Overview table provides a quick reference of the various laws and policies used throughout the world. Further details are available in the sections below the table.
